api/app/signals/apps/signals/models/expression.py

```diff
@@ -48,6 +48,7 @@ class Meta:
             ('sia_expression_read', 'Inzien van expressies'),
             ('sia_expression_write', 'Wijzigen van expressies'),
         )
+        ordering = ['pk']
```

Comment:
Forgot to run `makemigrations`?

There are now changes in the model that are not reflected in a migration. Please add the migration.

Reply:
will fix